### PR TITLE
fix: Self setup missing global classes

### PR DESCRIPTION
**Pull request by otDan**
- Adds all the missing classes for the self setup script

Mentioned in https://github.com/GodotModding/godot-mod-loader/issues/373

### CODE REVIEW

**Review by Qubus0 — APPROVED** [review]
are these the same as for godot 3? or more?

**Comment by otDan** [comment]
> are these the same as for godot 3? or more?

not everything is the same I believe which is why I have not pushed it to 3.x yet, will have to check better tomorrow

File: addons/mod_loader/mod_loader_setup.gd [hunk 1]
```diff
@@ -51,7 +101,7 @@ const new_global_classes := [
 		"language": "GDScript",
 		"path": "res://addons/mod_loader/api/mod.gd"
 	}, {
-		"base": "RefCounted",
+		"base": "Reference",
```
Comment (KANAjetzt) [inline addons/mod_loader/mod_loader_setup.gd:104]: Isn't `Reference` 3.x and `RefCounted` 4.x?

Comment (otDan) [reply addons/mod_loader/mod_loader_setup.gd:104]: it seems like I was testing with a 3.x game 🤦🏻

**Comment by otDan** [comment]
Actually tested version coming

**Pull request closed without merging**